webui/react/src/e2e/models/pages/ProjectDetails.ts

```diff
@@ -11,8 +11,10 @@ import { Pivot } from 'e2e/models/hew/Pivot';
  * @param {Page} page - The '@playwright/test' Page being used by a test
  */
 export class ProjectDetails extends BasePage {
-  readonly title: RegExp =
-    /(Uncategorized Experiments|Project Details) - (Determined|HPE Machine Learning Development Environment)/;
+  readonly title: RegExp = new RegExp(
```

Comment:
oh this is pretty annoying! 🤔

Reply:
I don't think it's too bad. We coulg have a getTitleRegEx like we do with url?

Reply:
or `get title` could always return regex? because we never need to visit it, it doesnt need to be a string